doc/nrf/gs_updating.rst

```diff
@@ -14,6 +14,18 @@ You might also want to switch to a newer release or check out the latest state o
 However, if you work with a specific release of the |NCS|, you do not need to update your repositories, because the release will not change.
 For an overview of changes in the latest releases, see :ref:`release_notes`.
 
+.. _gs_updating_in_vsc:
+
+Updating |VSC|
+*****************
+
+Visual Studio Code checks for extension updates and automatically installs them when they are available.
+After an extension is updated, VS Code prompts you to reload the application.
+If you disabled automatic updates, open the :guilabel:`Extensions` tab, locate the |VSC| extension, and click on the :guilabel:`Update` button if it is visible.
```

Comment:
I think there is just a little bit too much information in one sentence here, and it leaves it ambiguous what to do when the Update button isn't visible.
```suggestion
If you disabled automatic updates, open the :guilabel:`Extensions` tab and locate the |VSC| extension.
The :guilabel:`Update` button becomes visible when an update is available.
Click on the button to install the update.
```
Or something along those lines. 
I'm not quite happy with my own wording.

Reply:
Good point, I have updated this now.